src/test/04-Stress/10K_array.ck

```diff
@@ -0,0 +1,19 @@
+
+for( 0 => int i; i < foo.cap(); i++ ) {
+ foo[i] => c.freq;
+ .01::T => now;
+}
+<<<"success">>>;
```

Comment:
I'm pretty sure this has to do with allocating too much on the `reg` stack; sometimes we overflow and write onto memory outside the stack bounds.